README.md

```diff
@@ -61,14 +61,19 @@ while developing. This is a living document, so feel free to add any notes that
 
 ### Backend
 
-* Set `.additionalProperties(false)` for each schema defined with FluentSchema, to return a 400 bad request error if any
-additional properties not defined in the schema are passed in through the request.
-    * We are using [FluentSchema](https://github.com/fastify/fluent-schema) to validate backend requests. This is the
-    default validator for Fastify, our backend framework. FluentSchema uses ajv under the hood, and compiles to the
-    more standard JSON Schema.
+* API requests are validated using [Ajv](https://github.com/epoberezkin/ajv) and JSON schema's are defined with [FluentSchema](https://github.com/fastify/fluent-schema)
```

Comment:
Minor: formatting looks a bit off here, maybe you're adding the line breaks manually?

Reply:
Ah yeah I did that on purpose, thought it was better, similar to limiting max line-length in the code (to e.g. 150 chars...) But I'm not sure if this is a best practice for markdown files
The end result seemed the same when parsed so this means the original would also be more readable, seems like this is allowed https://stackoverflow.com/questions/29955688/in-markdown-is-there-an-official-best-practice-as-relates-to-line-length

Seems like some markdown renderers might have different behavior, but on Github it's ignored
see https://stackoverflow.com/questions/41513565/markdown-and-line-wrapping
so we could perhaps implement it as a best practice